uefi-test-runner/src/proto/string/unicode_collation.rs

```diff
@@ -0,0 +1,99 @@
+use core::cmp::Ordering;
+use uefi::prelude::*;
+use uefi::proto::string::unicode_collation::{StrConversionError, UnicodeCollation};
+use uefi::{CStr16, CStr8};
+
+pub fn test(bt: &BootServices) {
+    info!("Testing the Unicode Collation protocol");
+
+    if let Ok(handles) = bt.find_handles::<UnicodeCollation>() {
+        for handle in handles {
```

Comment:
I'm not sure if this makes sense.. but I'm not an expert with the uefi-API. Why would there be multiple handles to the same protocol? @GabrielMajeri  @nicholasbishop

Reply:
It looks like this protocol can have multiple instances (i.e. multiple handles). From section 21.1.1:
"One or more of the EFI_UNICODE_COLLATION_PROTOCOL instances may be present at one time.
Each protocol instance can support one or more language codes."